x-pack/plugins/features/server/oss_features.ts

```diff
@@ -46,7 +46,7 @@ export const buildOSSFeatures = ({ savedObjectTypes, includeTimelion }: BuildOSS
       }),
       icon: 'visualizeApp',
       navLinkId: 'kibana:visualize',
-      app: ['kibana'],
+      app: ['kibana', 'lens'],
```

Comment:
Technically unrelated to your changeset, but should the `dashboard` feature privileges also grant `read` access to the `lens` saved object type, like we do for `timelion-sheet` today?

Reply:
@wylieconlon can Lens visualizations currently be "embedded" in Dashboards?

Reply:
Even if Lens can be "embedded" in Dashboards, I'd prefer this separate bug be addressed in another PR.